src/overlays/actors/ovl_Obj_Switch/z_obj_switch.h

```diff
@@ -17,26 +17,21 @@ typedef enum {
 } ObjSwitchType;
 
 typedef enum {
-    /* 0 */ OBJSWITCH_SUBTYPE_FLOOR_0,
-    /* 1 */ OBJSWITCH_SUBTYPE_FLOOR_1,
-    /* 2 */ OBJSWITCH_SUBTYPE_FLOOR_2,
-    /* 3 */ OBJSWITCH_SUBTYPE_FLOOR_3
-} ObjSwitchSubTypeFloor;
-
-typedef enum {
-    /* 0 */ OBJSWITCH_SUBTYPE_EYE_0,
-    /* 1 */ OBJSWITCH_SUBTYPE_EYE_1
-} ObjSwitchSubTypeEye;
-
-typedef enum {
-    /* 0 */ OBJSWITCH_SUBTYPE_CRYSTAL_0,
-    /* 1 */ OBJSWITCH_SUBTYPE_CRYSTAL_1,
-    /* 4 */ OBJSWITCH_SUBTYPE_CRYSTAL_4 = 4
-} ObjSwitchSubTypeCrystal;
+            // Switches that can only be turned on (On -> Flag Set)
+    /* 0 */ OBJSWITCH_SUBTYPE_ONCE,
+            // Switches that can be turned on and off (On -> Flag Set, Off -> Flag Cleared)
+    /* 1 */ OBJSWITCH_SUBTYPE_TOGGLE,
+            // Floor Types only, released when not stood on (Down -> Flag Set, Up -> Flag Cleared)
+    /* 2 */ OBJSWITCH_SUBTYPE_RESET,
+            // Floor Types only, unused, inverted Switch Flag behavior (Down -> Flag Cleared, Up -> Flag Set)
+    /* 3 */ OBJSWITCH_SUBTYPE_RESET_INVERTED,
+            // Crystal Types only, syncs with the Switch Flag (On -> Flag Set, Off -> Flag Cleared)
+    /* 4 */ OBJSWITCH_SUBTYPE_SYNC
```

Comment:
I would put these comments beside the enum value
example
` /* 0 */ OBJSWITCH_SUBTYPE_ONCE,  // Switches that can only be turned on (On -> Flag Set)`

Reply:
I'm not 100% sure the formatting is good in the [commit](https://github.com/Yanis42/oot/commit/f8c15dab3225f89891591105825617571363c1e4) I did